data/frame_json.go

```diff
@@ -19,14 +19,22 @@ const simpleTypeNumber = "number"
 const simpleTypeBool = "bool"
 const simpleTypeTime = "time"
 
+type FrameJSONStyle int
+
+const (
+	WithSchmaAndData FrameJSONStyle = iota
```

Comment:
Another approach *may* be to have multiple type alias's
```
type FrameSchemaJSON Frame
type FrameDataJSON Frame
```
and register different encoders for them?